src/ClusterBootstrap/template/etcd/docker_etcd.sh

```diff
@@ -6,10 +6,16 @@ docker run -d -v /usr/share/ca-certificates/:/etc/ssl/certs -v /var/etcd:/var/et
  --restart always \
  --name etcd dlws/etcd:3.1.10 /usr/local/bin/etcd \
  -name $HOSTNAME \
- -advertise-client-urls http://{{cnf["etcd_node_ip"]}}:{{cnf["etcd3port1"]}} \
- -listen-client-urls http://0.0.0.0:{{cnf["etcd3port1"]}} \
- -initial-advertise-peer-urls http://{{cnf["etcd_node_ip"]}}:{{cnf["etcd3portserver"]}} \
- -listen-peer-urls http://0.0.0.0:2380 \
+ {% if cnf["etcd_node_num"] == 1 %}-initial-cluster {{cnf["hostname"]}}=https://{{cnf["etcd_private_ip"]}}:{{cnf["etcd3portserver"]}} \
+ -initial-cluster-state new \
+ -initial-cluster-token {{cnf["clusterId"]}} \
+ -advertise-client-urls https://{{cnf["etcd_private_ip"]}}:{{cnf["etcd3port1"]}} \
+ -listen-client-urls https://{{cnf["etcd_private_ip"]}}:{{cnf["etcd3port1"]}},https://127.0.0.1:{{cnf["etcd3port1"]}} \
+ -initial-advertise-peer-urls https://{{cnf["etcd_private_ip"]}}:{{cnf["etcd3portserver"]}} \
+ -listen-peer-urls https://{{cnf["etcd_private_ip"]}}:{{cnf["etcd3portserver"]}} \
```

Comment:
do you need to use "0.0.0.0" in "listen-client-urls" and "listen-peer-urls"?

Reply:
updated